configs/defs/centos.textproto

```diff
@@ -260,6 +261,7 @@ benchmark_configs: {
 benchmark_configs: {
   id: "selinux-installed"
   compliance_note: {
+    version: { cpe_uri: "cpe:/o:rockylinux:rockylinux:8.5" version: "1.0.0" benchmark_document: "CIS Rocky Linux 8" }
```

Comment:
If these checks apply to both CentOS and RockyLinux then they're not really specific to CentOS - I think we should move them into the generic_linux.textproto

Reply:
Should I consider a scan generic even if the remediation is not? For example, a check on installed software will look for files in paths common to most Linux distribution, but the remediation will say something like use yum to uninstall.

Reply:
Hm, is the common denominator between the shared RocyLinux and CentOS checks that they both use yum? In that case we could create a separate definition file for yum-based distros.

Reply:
I have moved the benchmarks to more generic files, let me know if this is what you had in mind!